hlsp.go

```diff
@@ -1,6 +1,30 @@
-package hlsp
+package parser
+
+import "fmt"
 
 // Parse receives either a YAML or JSON AsyncAPI document. It parses the document and checks if it's valid AsyncAPI. Skips specification extensions and schemas validation. If validation fails, the Parser/Validator should trigger an error. Produces a beautified version of the document in JSON Schema Draft 07
-func Parse() string {
-	return "Hello World"
+func Parse(document string) (string, err error) {
+
+	return
+}
+
+func convertToJSON(doc string) (string, error) {
+	if isJSON(doc) {
```

Comment:
Is it important to know if the input doc is in JSON format? JSON is a subset of YAML.